src/commonTest/resources/dev/rhovas/interpreter/programs/rosettacode/Fibonacci.rho

```diff
@@ -0,0 +1,26 @@
+func pfib(n: Integer): Integer {
+    require n >= 0;
+
+    var lastFib = -1;
+    var fib = 1;
+    range(0, n, :incl).for {
+        var sum = fib + lastFib;
+        lastFib = fib;
+        fib = sum;
+    };
+    return fib;
+}
+
+func fib(n: Integer): Integer {
+    match ([n < 0, n.mod(2)]){
```

Comment:
I'd use a conditional match here since these cases don't overlap.

```scala
match {
    n >= 0: return fibonacci(n);
    n.mod(2) == 0: return -fibonacci(-n);
    else: return fibonacci(-n);
}
```

Rhovas is missing some language and standard library features to do this; but in the future here might be an alternative idea for how to do this:

```scala
fibonacci(n.abs()).match {
    val < 0 && val.mod(2) == 0: return -val;
    else: return val;
}
```